### PR TITLE
[Email routing] Email Workers

### PR DESCRIPTION
**Pull request by marciocloudflare**
Addresses PCX-4426

### CODE REVIEW

File: content/email-routing/_partials/_enable-create-worker.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+4. (Optional) Enter a descriptive Email Worker name in **Create a worker name**. 
+
+5. In **Select a starter**, select the starter template that best suits your needs. You can also start from scratch and build your own Email Worker with **Create my own**. After choosing your template, select **Create**.
+
+6. Now, configure your code on the left side of the screen. For example, if you are creating an Email Worker from the Allowlist template:
+	1. In `const allow = ["friend@example.com", "coworker@example.com"];` replace the email examples with the addresses you want to allow emails from.
+	2. In `await message.forward("inbox@corp");` replace the email address example with the address where emails should be forwarded to.
+
+7. (Optional) You can test your logic on the right side of the screen. In the **From** field enter either an email address from your approved senders list or one that is not on the approved list. When you select **Trigger email event** you should see a message telling you if the email address is allowed or rejected.
```
Comment (kodster28) [inline content/email-routing/_partials/_enable-create-worker.md]:
```suggestion
7. (Optional) You can test your logic on the right side of the screen. In the **From** field, enter either an email address from your approved senders list or one that is not on the approved list. When you select **Trigger email event** you should see a message telling you if the email address is allowed or rejected.
```

File: content/email-routing/email-workers/_index.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+pcx_content_type: concept
+layout: single
+weight: 4
+---
+
+{{<beta>}}Email Workers{{</beta>}}
+
+With Email Workers you can leverage the power of Cloudflare Workers to implement any logic you need to process your emails and create complex rules. These rules determine what happens when you receive an email.
+
+Creating your own rules with Email Workers is as easy or complex as you want. You can begin using one of the starter templates that are pre-populated with code for popular use-cases. These templates allow you to create a blocklist, allowlist or send notifications to Slack. Advanced users can also write their own code, and skip the starter templates.
```
Comment (kodster28) [inline content/email-routing/email-workers/_index.md]:
```suggestion
Creating your own rules with Email Workers is as easy or complex as you want. You can begin using one of the starter templates that are pre-populated with code for popular use-cases. These templates allow you to create a blocklist, allowlist, or send notifications to Slack.
```

Comment (kodster28) [reply content/email-routing/email-workers/_index.md]: Duplicated in sentence following

File: content/email-routing/email-workers/_index.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+}
+```
+
+Refer to the [Workers Languages](/workers/platform/languages/) for more information regarding the languages you can use with Workers.
+
+## How to use Email Workers
+
+To use Email Routing with Email Workers there are three steps involved:
+
+1. Creating the Email Worker.
```
Comment (kodster28) [inline content/email-routing/email-workers/_index.md:38]: This is fine as is, but wanted to highlight you could also add a mermaid diagram here if you wanted for a more visual approach.

Probably not worth doing now so you can close out the PR, but just a reminder that it's an available option.

**Review by kodster28 — APPROVED** [review]

File: content/email-routing/email-workers/_index.md (hunk 3 above)
Comment (marciocloudflare) [reply content/email-routing/email-workers/_index.md:38]: Oh, good one. Totally forgot about mermaid. I'll take a note and replace this in a future PR :D

**Pull request merged**